src/Engine/RadiumEngine.cpp

```diff
@@ -43,7 +43,7 @@ RadiumEngine::~RadiumEngine() = default;
 
 void RadiumEngine::initialize() {
     LOG( logINFO ) << "*** Radium Engine ***";
-    m_resourcesRootDir    = {Core::Resources::getRadiumResourcesDir()};
+    m_resourcesRootDir    = {Core::Resources::getRadiumResourcesPath()};
```

Comment:
As the result of this is used quite several time (and not only at initialization), why not caching it instead of executing `search_path` each time ?

Reply:
Not the purpose of this PR.